dali/pipeline/data/dltensor.cc

```diff
@@ -40,14 +41,14 @@ void DLManagedTensorDeleter(DLManagedTensor *self) {
 void DLMTensorPtrDeleter(DLManagedTensor* dlm_tensor_ptr) {
   if (dlm_tensor_ptr) {
```

Comment:
Shouldn't it be
```suggestion
  if (dlm_tensor_ptr && dlm_tensor_ptr->deleter) {
```
?